driver/src/orderbook/mod.rs

```diff
@@ -29,6 +29,11 @@ pub trait StableXOrderBookReading {
     /// # Arguments
     /// * `batch_id_to_solve` - the index for which returned orders should be valid
     fn get_auction_data(&self, batch_id_to_solve: U256) -> Result<(AccountState, Vec<Order>)>;
+    /// Perform potential heavy initialization of the orderbook. If this fails or wasn't called
+    // the orderbook will initialize on first use of `get_auction_data`.
+    fn initialize(&self) -> Result<()> {
```

Comment:
Should implementors of OrderbookReading rely on this method being called before the first invocation or is it more of a non-guaranteed optimization (like warming a cache)? In the latter case, we would probably want to call it something like prepare or so.

I don't feel strongly either way, but could we clarify the expected behavior in the docs?

Reply:
I think the docs mention that it doesn't have to be called:
> `If this fails or wasn't called the orderbook will initialize on first use of get_auction_data.`

Reply:
🙈